vizro-core/docs/pages/user-guides/actions.md

```diff
@@ -17,7 +17,7 @@ To attach an action to a component, you must enter the [`Action`][vizro.models.A
 add a desired pre-defined action function into the `function` argument of the [`Action`][vizro.models.Action].
 
 ??? note "Note on `Trigger`"
-    Currently each component has one pre-defined trigger property. A trigger property is an attribute of the component that triggers a configured action (e.g. for the `Button` it is `n_click`).
+Currently each component has one pre-defined trigger property. A trigger property is an attribute of the component that triggers a configured action (e.g. for the `Button` it is `n_click`).
```

Comment:
I think this actually needs to stay as previously as otherwise the note is not rendered correctly:

![Screenshot 2024-02-19 at 19 48 07](https://github.com/mckinsey/vizro/assets/90609403/8ef1eddf-5fd0-4405-b6ab-2392387b7d8e)